lib/matplotlib/_mathtext.py

```diff
@@ -31,18 +31,13 @@
 from .font_manager import FontProperties, findfont, get_font
 from .ft2font import FT2Font, FT2Image, Kerning, LoadFlags
 
-from packaging.version import parse as parse_version
-from pyparsing import __version__ as pyparsing_version
-if parse_version(pyparsing_version).major < 3:
-    from pyparsing import nestedExpr as nested_expr
-else:
-    from pyparsing import nested_expr
+from pyparsing import nested_expr
```

Comment:
This can be inlined to the above import.

Reply:
👍 done